### ISSUE TITLE
docs: Document how to export metrics to prometheus / otel

### ISSUE THREAD

**Issue by mauriciovasquezbernal** [inspektor-gadget/inspektor-gadget#3306]
https://www.inspektor-gadget.io/docs/latest/gadget-devel/metrics contains information about how to add metrics to a gadget, however we're missing the documentation for users to setup everything to export metrics from existing gadgets.

**Comment by larvinloy** [comment]
Is there an example or test config in the repo that could inform me on how to set up the otel-metrics for a gadget?

**Comment by larvinloy** [comment]
Yes, I had a look and while that works, I was looking for otel-metrics operator way of doing it so that I could eventually plug the metrics into our datadog agent. 

I tried setting the otel config in the `gadget` configmap, but I think that the daemonset is not picking it up.
```
otel-metrics:                                                                                                                                                                                                                                                                                                        
  otel-metrics-listen: true
```

**Comment by mauriciovasquezbernal** [comment]
Perhaps @flyth can help you with this one.

**Comment by flyth** [comment]
> Perhaps @flyth can help you with this one.

I started drafting something [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/d1630401a41a67a762b2c4acee2be2de6181e3e7/docs/reference/export-metrics.mdx) (#3539). That's however only valid for image-based gadgets

**Comment by mauriciovasquezbernal** [comment]
Fixed by #3539
